omnibot/processor.py

```diff
@@ -311,23 +311,27 @@ def _handle_post_message(message, kwargs):
     except json.decoder.JSONDecodeError:
         logger.exception('JSON decode failure when parsing {}'.format(kwargs))
         return
-    logger.debug(ret, extra=message.event_trace)
-    if not ret['ok']:
+    except:
```

Comment:
bare excepts also catch signals. you should catch SlackClientError as it's their generic exception

Reply:
updated